lms/static/js/instructor_dashboard/student_admin.js

```diff
@@ -474,7 +476,10 @@
                 success: this.clear_errors_then(function() {
                     return alert(fullSuccessMessage);  // eslint-disable-line no-alert
                 }),
-                error: statusAjaxError(function() {
+                error: statusAjaxError(function(response) {
+                    if (response.responseText === 'Specified module does not support rescoring.') {
```

Comment:
No. An error is an eror and returning a 400 in this case follows the HTTP standard.

My comment is about the usage of the server-returned error string.  You can display it directly to the user to surface the underlying issue.  Otherwise, if you don't want the server to provide the user-displayed error string, then have it return an error code.